class-parsely-recommended-widget.php

```diff
@@ -103,6 +103,11 @@ function widgetLoad() {
 					if (parentDiv.length === 0) {
 						parentDiv = jQuery.find('.Parsely_Recommended_Widget');
 					}
+					
+					// make sure widget is not already on page
+					if (jQuery.find("div.parsely-recommendation-widget").length != 0) {
```

Comment:
I think you actually want `parentDiv.find()` instead of `jQuery.find()`. Because we only care about not duplicating the widget within a parent div where it already exists. But it's totally ok (and expected by some customers) to have multiple instances of the widget in different locations on the page. `jQuery.find()` will do a global page search and just return the first element it finds.